tests/src/test/java/io/streamnative/pulsar/handlers/mqtt/base/MQTTTestBase.java

```diff
@@ -22,12 +22,32 @@
 import org.fusesource.mqtt.client.MQTT;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
 
 /**
  * Base test class for MQTT Client.
  */
 @Slf4j
 public class MQTTTestBase extends MQTTProtocolHandlerTestBase {
+
+    public static final int TIMEOUT = 60 * 1000;
+
+    @DataProvider(name = "mqttTopicNames")
+    public Object[][] mqttTopicNames() {
+        return new Object[][] {
+                { "public/default/t0" },
```

Comment:
It seems that the local name `public/default/t0` may cause confusion. The name `public/default` is not the namespace, right?

Reply:
Yes, changed the topic to avoid confusing.